salt/modules/mssql.py

```diff
@@ -250,23 +252,69 @@ def login_exists(login, **kwargs):
         return 'Could not find the login: {0}'.format(e)
 
 
-def user_exists(username, **kwargs):
+def login_create(login, new_login_password=None, new_login_domain=None, new_login_options=[], **kwargs):
```

Comment:
We cannot have mutable values like lists as default keyword arguments. Please change this to something like `None` and then transform it to a list inside the function if necessary.